src/app-layer-ftp.h

```diff
@@ -115,6 +116,7 @@ typedef struct FtpState_ {
     uint8_t *input;
     int32_t input_len;
     uint8_t direction;
+    uint8_t active;
```

Comment:
use 'bool' instead? Plus then true/false instead of 1/0

Reply:
Will do.